src/main/java/org/tugraz/sysds/conf/DMLConfig.java

```diff
@@ -88,7 +88,8 @@
 	public static final String PRINT_GPU_MEMORY_INFO = "sysds.gpu.print.memoryInfo";
 	public static final String EVICTION_SHADOW_BUFFERSIZE = "sysds.gpu.eviction.shadow.bufferSize";
 	
-	public static final String DEFAULT_FEDERATED_PORT = "25501";
+	public static final String DEFAULT_FEDERATED_PORT = "4040"; // borrowed default Spark Port
+	public static final String DEFAULT_NUMBER_OF_FEDERATED_WORKER_THREADS = "10";
```

Comment:
Since the port i originally chose, was not open by default in default cluster setup, i changed it to one that is known to be open in most cases for a cluster setup.

Other options are available, Link: [OpenPorts](https://www.cloudera.com/tutorials/hortonworks-sandbox-guide/3.html)
But while trying one of the "Port for custom use" (specificly 111) It did not work.